code/modules/reagents/recipes/recipes_food_drinks.dm

```diff
@@ -96,6 +96,15 @@
 	required_reagents = list("tomatojuice" = 2, "water" = 1, "sugar" = 1)
 	result_amount = 4
 
+/datum/chemical_reaction/vinegar
+	result = "vinegar"
+	required_reagents = list("wine" = 1, "water" = 1, "flour" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/barbecue
```

Comment:
No salts?

Reply:
Making vinegar involves a "mother base" (I'm using flour as a substitute for yeast/vinegar mother/etc), wine, and all dilluted in a bit of water, it doesn't take salt to make!